src/mixins/meta.ts

```diff
@@ -61,15 +80,23 @@ export function PageMetaI18nMixin(
         metaI18nTitle: string;
         metaI18nDescription: string;
         metaI18nRoutePath: string;
+        metaI18nAlternateLocale?: QMetaI18nAlternate;
       }
     ) {
+      // Parses options
+      const alternate = this.metaI18nAlternateLocale;
+      let alternateMeta = pageAlternateLocales(alternate?.locales ?? [], (options?.alternate.openGraph && alternate?.currentLocale) ? alternate.currentLocale : undefined );
```

Comment:
Qua bisogna rendere il codice un po' più leggibile, lavorando con variabili intermedie e naming.
La confusione è generata soprattutto dal fatto che le opzioni hanno un "alternate", ma anche localmente c'è "alternate" come variabile locale

Reply:
Updated names of the variables and removed the optional boolean flag which allowed to specify if the locale needed to be activate or not.